tests/gitlab_test.go

```diff
@@ -201,6 +201,7 @@ func TestGitlabRunnerRoleECR(t *testing.T) {
 	aws.CreateECRRepo(t, region, reponame)
```

Comment:
Should you be checking for an error here? Also, you can defer deleteECRRepo like https://github.com/gruntwork-io/terratest/blob/master/modules/aws/ecr_test.go#L15 - that might be cleaner.

Reply:
No need to check for error.  If it fails to create the repo, it fails the test.

The defer is interesting.  I'll have to think about whether that will help in some way.  I think that the logic is sound here as is right now though.  If there was a segfault or the test was killed, we need to clean the orphaned repo up before we test whether we can do a create.

Reply:
Ah, I see. https://github.com/gruntwork-io/terratest/blob/master/modules/aws/ecr.go#L16 wraps the inner CreateECRRepo and checks for the error there.

A defer would just save you a call to GetECRRepoE in the cleanup, and I think would make it a bit more future-proof if we add code that has a chance to panic or exit early. But I defer to you, since you're actually writing the code ;-)

LGTM